packages/core/useClipboard/index.md

```diff
@@ -18,9 +18,17 @@ const { text, copy, copied, isSupported } = useClipboard({ source })
 ```
 
 ```html
-<button @click='copy()'>
-  <!-- by default, `copied` will be reset in 1.5s -->
-  <span v-if='!copied'>Copy</span>
-  <span v-else>Copied!</span>
-</button>
+ <div v-if="isSupported">
+    <button @click='copy(source)'>
+      <!-- by default, `copied` will be reset in 1.5s -->
+      <span v-if='!copied'>Copy</span>
+      <span v-else>Copied!</span>
+    </button>
+    <p>
+      Current copied: <code>{{ text || 'none' }}</code>
+    </p>
+  </div>
+  <p v-else>
+    Your browser does not support Clipboard API
+  </p>
```

Comment:
Please fix the format (indent)

Reply:
Is it that you want the indents removed or you want a two space indent?

Reply:
Should be two spaces per indent